src/symbol/graph_executor.cc

```diff
@@ -881,8 +884,9 @@ void GraphExecutor::Print(std::ostream &os) const {
       os << '\n';
     }
   }
-  os << "Total " << (total_allocated_bytes_ >> 20UL) <<" MB allocated\n";
+  os << "Total " << (total_allocated_bytes_ >> 20UL) <<" MiB allocated\n";
   os << "Total " << total_allocated_temp_ <<" TempSpace resource requested\n";
+  os << "Additional " << (memory_used >> 20UL) <<" MiB Memory used\n";
```

Comment:
consider put it before the first line, some of the scripts in memory monger relies on the fact that total memory estimation appears in the certain line https://github.com/dmlc/mxnet-memonger/blob/master/memonger.py#L118